### PR TITLE
Undefined method 'format_errors' when db:seed fail

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
[test foreman]
Seems like only transient failures, but lets see

**Review by ezr-ondrej — COMMENTED** [review]
You're removing the helper method, but plugins are still using it, should we update those first?
https://github.com/search?q=org%3Atheforeman+format_errors&type=code

File: db/seeds.rb [hunk 1]
```diff
@@ -5,10 +5,5 @@
 #
 # Please ensure that all templates are submitted to community-templates, then they will be synced in.
 
-# define all helpers here
-def format_errors(model = nil)
-  SeedHelper.format_errors(model)
```
Comment (ezr-ondrej) [inline db/seeds.rb:10]: To get this in quickly, should we have a deprecation warning for next version and keep it until the next version?

**Comment by stejskalleos** [comment]
@ezr-ondrej I reverted the removal of the `format_errors` since it's used in many plugins.
I kept the changes for core seed files, where the `format_errors` helper is not available.

**Comment by ezr-ondrej** [comment]
@stejskalleos could we deprecate the method in core seed files? It is quite a bad practice, so we should remove it in the future.

**Review by ezr-ondrej — APPROVED** [review]
LGTM, let see what Jenkins thinks :)

**Comment by ezr-ondrej** [comment]
[test foreman]
[test katello]

**Pull request merged**